packages/cozy-client/src/CozyClient.js

```diff
@@ -1102,6 +1103,27 @@ class CozyClient {
     return id
   }
 
+  /**
+   * getInstanceOptions - Returns a set of instance options (such as domain or current app slug) that has been previously loaded
+   *
+   * @returns {object}
+   */
+  getInstanceOptions() {
+    return this.instanceOptions
+  }
+
+  /**
+   * loadInstanceOptionsFromDOM - Loads the dataset injected by the Stack in web pages and exposes it through getInstanceOptions
+   *
+   * @param {string} [selector=[role=application]] A selector for the node that holds the dataset to load
+   *
+   * @returns {void}
+   */
+  loadInstanceOptionsFromDOM(selector = '[role=application]') {
+    const root = document.querySelector(selector)
+    this.instanceOptions = { ...root.dataset } // convert from DOMStringMap to plain object
+  }
```

Comment:
Do you think it could be done when instantiating the client ? This way we do not have to remember to do it ?

Reply:
Yeah, I thought about that after I submitted the PR actually. I think that would be a good idea, i'll add it.